### ISSUE TITLE
TypeError: t is not a function when running any serverless command

### ISSUE THREAD

**Issue by rcoundon** [serverless/serverless#8393]
When running any sls command deploy/start a TypeError is thrown t: is not a function

```
TypeError: t is not a function
      at Function.v.extend (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/dayjs/dayjs.min.js:1:6318)
      at Object.<anonymous> (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/lib/plugins/aws/metrics/awsMetrics.js:11:7)
      at Module._compile (internal/modules/cjs/loader.js:1137:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
      at Module.load (internal/modules/cjs/loader.js:985:32)
      at Function.Module._load (internal/modules/cjs/loader.js:878:14)
      at Module.require (internal/modules/cjs/loader.js:1025:19)
      at require (internal/modules/cjs/helpers.js:72:18)
      at Object.<anonymous> (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/lib/plugins/index.js:31:3)
      at Module._compile (internal/modules/cjs/loader.js:1137:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
      at Module.load (internal/modules/cjs/loader.js:985:32)
      at Function.Module._load (internal/modules/cjs/loader.js:878:14)
      at Module.require (internal/modules/cjs/loader.js:1025:19)
      at require (internal/modules/cjs/helpers.js:72:18)
      at PluginManager.loadAllPlugins (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/lib/classes/PluginManager.js:127:9)
      at /Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/lib/Serverless.js:88:39
      at tryCatcher (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/util.js:16:23)
      at Promise._settlePromiseFromHandler (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:547:31)
      at Promise._settlePromise (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:604:18)
      at Promise._settlePromise0 (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:649:10)
      at Promise._settlePromises (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:729:18)
      at Promise._fulfill (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:673:18)
      at Promise._settlePromise (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:617:21)
      at Promise._settlePromise0 (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:649:10)
      at Promise._settlePromises (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:729:18)
      at Promise._fulfill (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:673:18)
      at Promise._resolveCallback (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:466:57)
      at Promise._settlePromiseFromHandler (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:559:17)
      at Promise._settlePromise (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:604:18)
      at Promise._settlePromise0 (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:649:10)
      at Promise._settlePromises (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:729:18)
      at Promise._fulfill (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:673:18)
      at PromiseArray._resolve (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise_array.js:127:19)
      at PromiseArray._promiseFulfilled (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise_array.js:145:14)
      at Promise._settlePromise (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:609:26)
      at Promise._settlePromise0 (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:649:10)
      at Promise._settlePromises (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:729:18)
      at Promise._fulfill (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:673:18)
      at Promise._resolveCallback (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:466:57)
      at Promise._settlePromiseFromHandler (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:559:17)
      at Promise._settlePromise (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:604:18)
      at Promise._settlePromise0 (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:649:10)
      at Promise._settlePromises (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:729:18)
      at Promise._fulfill (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:673:18)
      at PropertiesPromiseArray.PromiseArray._resolve (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise_array.js:127:19)
      at PropertiesPromiseArray._promiseFulfilled (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/props.js:78:14)
      at Promise._settlePromise (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:609:26)
      at Promise._settlePromise0 (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:649:10)
      at Promise._settlePromises (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:729:18)
      at _drainQueueStep (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/async.js:93:12)
      at _drainQueue (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/async.js:86:9)
      at Async._drainQueues (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/async.js:102:5)
      at Immediate.Async.drainQueues [as _onImmediate] (/Users/rcoundon/.nvm/versions/node/v12.18.4/lib/node_modules/serverless/node_modules/bluebird/js/release/async.js:15:14)
      at processImmediate (internal/timers.js:456:21)
```

This could be down to changes in dayjs between 1.9.1 and 1.9.2 as the package.json in serverless uses ^1.9.1

Commenting out line 11 of serverless/lib/plugins/aws/metrics/awsMetrics.js allow the process to run:
```js
dayjs.extend(LocalizedFormat);
```

<details>
<summary><code>serverless.yml</code></summary>

```yaml
service: someService
frameworkVersion: '>2.0.0 <=2.6.0'
org: someOrg

package:
  individually: true

provider:
  name: aws
  stage: dev
  runtime: nodejs12.x
  versionFunctions: false
  memorySize: 512
  timeout: 10
  region: ${opt:region, self:custom.defaultRegion}
  account: ${opt:account, self:custom.account}

```

</details>

<details>
<summary><b><code></code> output</b></summary>

```
sls --version
sls deploy

...any sls command
```
</details>

<b>Installed version</b>

```
  Your Environment Information ---------------------------
     Operating System:          darwin
     Node Version:              12.18.4
     Framework Version:         2.6.0
     Plugin Version:            4.0.4
     SDK Version:               2.3.2
     Components Version:        3.2.1
```

**Comment by medikoo** [comment]
@rcoundon thanks for reporting. it was a regression in one of the dependencies we use.

See https://github.com/iamkun/dayjs/issues/1132

It's already fixed, ensure to update to latest version

**Comment by Ali-Dalal** [comment]
already fixed

**Comment by rcoundon** [comment]
Ok, thanks. Sorry, latest version of serverless? if so, I'm already running 2.6.0 and that appears to be latest?

**Comment by rcoundon** [comment]
Sorry, ready the linked issue, dayjs. Got it

**Comment by ronaldndirangu** [comment]
I am experiencing the same issue how did you fix it @rcoundon ?

**Comment by rcoundon** [comment]
@ronaldndirangu - I just reinstalled serverless. On reinstallation it'll pick up v1.9.3 of dayjs which fixes the issue.

**Comment by medikoo** [comment]
Closing as it was fixed at origin, and also we've released v2.7.0 which guarantees to take updated version
